src/ext/FirewallExtension/ca/firewall.cpp

```diff
@@ -6,7 +6,7 @@ LPCWSTR vcsFirewallExceptionQuery =
     L"SELECT `Name`, `RemoteAddresses`, `Port`, `Protocol`, `Program`, `Attributes`, `Profile`, `Component_`, `Description` FROM `WixFirewallException`";
 enum eFirewallExceptionQuery { feqName = 1, feqRemoteAddresses, feqPort, feqProtocol, feqProgram, feqAttributes, feqProfile, feqComponent, feqDescription };
 enum eFirewallExceptionTarget { fetPort = 1, fetApplication, fetUnknown };
-enum eFirewallExceptionAttributes { feaIgnoreFailures = 1 };
+enum eFirewallExceptionAttributes { feaIgnoreFailures = 0x1, feaOutbound = 0x42 };
```

Comment:
feaOutbound should be 0x2 since it should only be a single bit, not two.

Reply:
Fixed